web/src/common/ErrorBoundary.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, ErrorInfo, ReactNode } from 'react'
+import { Typography, Icon } from '@equinor/eds-core-react'
+import { account_circle } from '@equinor/eds-icons'
+
+interface Props {
+  children?: ReactNode
+}
+
+interface State {
+  hasError: boolean
+}
+
+class ErrorBoundary extends Component<Props, State> {
```

Comment:
Ah, I see, it can't be done otherwise at the moment. There is a [npm package](https://github.com/bvaughn/react-error-boundary) that could be used, but it doesn't seem to be properly maintained, so I suppose this is the best option.